### ISSUE TITLE
Layout View: Grant a place of honor to the "Layout Algorithm" option

### ISSUE THREAD

**Issue by spoenemann** [eclipse/elk#34]
The option "Layout Algorithm" is the most important one and has a special meaning among all layout options. It should be put into a privileged place, e.g. a dedicated line above the property sheet that contains the other options.

**Comment by le-cds** [comment]
Yeah, why not. That line should of course only be visible (or at least only enabled) if a diagram element is selected that is a parent.
